src/schema.js

```diff
@@ -183,10 +183,27 @@ const authTokenSchema = Joi.object({
 	}
 })
 
+
+const createGifsSchema = Joi.object({
```

Comment:
let's use a singular name `Gif` not `Gifs`

Reply:
yes please